src/helm.exec.ts

```diff
@@ -173,6 +173,26 @@ export function helmGet(resourceNode: explorer.ResourceNode) {
     });
 }
 
+// helmPackage runs the Helm package on a chart within your project.
+export function helmPackage() {
+    pickChart((path) => {
+        vscode.window.showInputBox({
+            prompt: "package path",
+            placeHolder: "package path"
+        }).then((ppath) => {
+            const packagepath = ppath;
```

Comment:
Not sure of the value of capturing `ppath` into a separate constant - could we not use it directly?  (In which case we would want to use the full name `packagePath`.)

Reply:
`packagePath` is needed to define a custom path where you want to save the chart packages.

Reply:
I understand that, I was trying to say that I don't think we need separate `ppath` and `packagePath` variables.  Can we not just write `....then((packagePath) => { … });`?